projects/packages/my-jetpack/_inc/components/my-jetpack-screen/styles.module.scss

```diff
@@ -1,26 +1,38 @@
+@import '../../../node_modules/@wordpress/base-styles/breakpoints';
+@import '../../../node_modules/@wordpress/base-styles/mixins';
+
 .heading {
 	margin-bottom: 0;
 }
 
 .notice {
+	box-sizing: border-box;
 	margin: 0;
+	padding: 12px 20px 12px 14px;
 	font-size: 16px;
+	line-height: 22px;
+	color: var( --jp-gray-80 );
+	border: 1px solid #DCDCDE;
+	border-left: 6px solid var( --jp-red-50 );
+	border-radius: 4px;
+	box-shadow: 0 4px 8px rgba(0, 0, 0, 0.03), 0 1px 2px rgba(0, 0, 0, 0.06);
 
 	// notice itself on error status
 	&:global(.is-error) {
-		background-color: var(--jp-red-0);
+		background-color: var( --jp-white );
 	}
 
 	// notice content
 	& :global(.components-notice__content) {
 		display: flex;
-		align-items: center;
 		margin: 0;
 		padding: 12px 4px;
+		flex-direction: column;
+		align-items: start;
 
-		// left icon
-		& > svg {
-			fill: var(--jp-red-60);
+		@include break-medium {
```

Comment:
Looks like this breakpoint should not match exactly the ones that we implement, since you're getting the ones from wordpress and not jetpack.

PbJqbW-7N-p2

Normally for this case, we prefer to insert a different class that handle this, and insert it using the `useBreakpointMatch`.

Example: https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/social/src/js/components/icon-text/index.js